.github/workflows/ci_backend.yml

```diff
@@ -0,0 +1,39 @@
+name: 'CI - Backend'
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+      - 'releases/**'
+    paths:
+      - 'RetrospectiveExtension.Backend/**'
+  pull_request:
+    branches:
+      - master
+      - dev
+      - 'releases/**'
+    paths:
+      - 'RetrospectiveExtension.Backend/**'
+  workflow_dispatch:
```

Comment:
@phongcao  Question: is workflow_dispatch a temporary addon or do you see it continue to exist?

Reply:
Good question. I'm afraid that it needs to be there at least until we've finished all CI/CD features and are ready to merge back to the main repo. We'll ask @polatengin if he wants to keep it for the future or not.